chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/listing/Headings.java

```diff
@@ -0,0 +1,228 @@
+    DEVELOPER_WEBSITE("VENDOR_WEBSITE__C", "VENDOR_WEBSITE_C", "VENDOR_WEBSITE", "DEVELOPER_WEBSITE__C",
+            "DEVELOPER_WEBSITE_C", "DEVELOPER_WEBSITE"),
+    SELF_DEVELOPER("SELF_DEVELOPER__C", "SELF_DEVELOPER_C", "SELF_DEVELOPER", "SELF-DEVELOPER"),
+    DEVELOPER_EMAIL("VENDOR_EMAIL__C", "VENDOR_EMAIL_C", "VENDOR_EMAIL", "DEVELOPER_EMAIL__C",
+            "DEVELOPER_EMAIL_C", "DEVELOPER_EMAIL"),
+    DEVELOPER_PHONE("VENDOR_PHONE__C", "VENDOR_PHONE_C", "VENDOR_PHONE", "DEVELOPER_PHONE__C",
+            "DEVELOPER_PHONE_C", "DEVELOPER_PHONE"),
+    DEVELOPER_CONTACT_NAME("VENDOR_CONTACT_NAME__C", "VENDOR_CONTACT_NAME_C", "VENDOR_CONTACT_NAME",
+            "DEVELOPER_CONTACT_NAME__C", "DEVELOPER_CONTACT_NAME_C", "DEVELOPER_CONTACT_NAME"),
+    TARGETED_USERS("Developer-Identified Target Users", "Developer_Identified_Target_Users"),
+    QMS_STANDARD_NAME("QMS Standard", "QMS_STANDARD"),
+    QMS_STANDARD_APPLICABLE_CRITERIA("QMS Standard Applicable Criteria"),
+    QMS_MODIFICATION("QMS Modification Description"),
+    ICS("ICS"),
+    ICS_SOURCE("ICS Source"),
+    ACCESSIBILITY_CERTIFIED("Accessibility Certified"),
+    ACCESSIBILITY_STANDARD("Accessibility Standard"),
+    K_1_URL("170.523(k)(1) URL", "170.523 (k)(1) URL"),
+    CQM_NUMBER("CQM Number", "CQM_NUMBER"),
+    CQM_VERSION("CQM Version", "CQM_VERSION"),
+    CQM_CRITERIA("CQM Criteria", "CQM_CRITERIA"),
+    SED_REPORT_URL("SED Report Hyperlink", "SED Report URL", "SED_REPORT_URL"),
+    SED_INTENDED_USERS("Description of the Intended Users"),
+    SED_TESTING_DATE("Date SED Testing was Concluded"),
+    PARTICIPANT_ID("Participant Identifier"),
+    PARTICIPANT_GENDER("Participant Gender"),
+    PARTICIPANT_AGE("Participant Age"),
+    PARTICIPANT_EDUCATION("Participant Education"),
+    PARTICIPANT_OCCUPATION("Participant Occupation/Role", "Participant Occupation", "Participant Role"),
+    PARTICIPANT_PROFESSIONAL_EXPERIENCE("Participant Professional Experience"),
+    PARTICIPANT_COMPUTER_EXPERIENCE("Participant Computer Experience"),
+    PARTICIPANT_PRODUCT_EXPERIENCE("Participant Product Experience"),
+    PARTICIPANT_ASSISTIVE_TECH("Participant Assistive Technology Needs"),
+    TASK_ID("Task Identifier"),
+    TASK_DESCRIPTION("Task Description"),
+    TASK_SUCCESS_MEAN("Task Success - Mean (%)", "Task Success Mean"),
+    TASK_SUCCESS_STDDEV("Task Success - Standard Deviation (%)", "Task Success Standard Deviation"),
+    TASK_PATH_DEV_OBS("Task Path Deviation - Observed #", "Task Path Deviation Observed"),
+    TASK_PATH_DEV_OPT("Task Path Deviation - Optimal #", "Task Path Deviation Optimal"),
+    TASK_TIME_MEAN("Task Time - Mean (seconds)", "Task Time Mean"),
+    TASK_TIME_STDDEV("Task Time - Standard Deviation (seconds)", "Task Time Standard Deviation"),
+    TASK_TIME_DEV_OBS("Task Time Deviation - Observed Seconds", "Task Time Deviation - Mean Observed Seconds",
+            "Task Time Deviation Observed"),
```

Comment:
Good candidate for the "don't split the line, we're ignoring line lengths", though not required